ui/src/app/pages/admin/admin-institution-edit.spec.tsx

```diff
@@ -44,39 +45,79 @@ describe('AdminInstitutionEditSpec', () => {
       .toContain('Display name must be 80 characters or less');
   });
 
-  it('should default DUA to Master', async() => {
+  it('should default DOMAINS to ADDRESSES in Registered Tier requirement', async() => {
     urlParamsStore.next({
       institutionId: 'Verily'
     });
     const wrapper = component();
     await waitOneTickAndUpdate(wrapper);
     expect(wrapper).toBeTruthy();
 
-    const agreementTypeDropDown = wrapper.find('[data-test-id="agreement-dropdown"]');
-    expect(agreementTypeDropDown.first().props().value).toBe(InstitutionMembershipRequirement.DOMAINS);
+    const rtAgreementTypeDropDown = wrapper.find('[data-test-id="rt-agreement-dropdown"]');
+
+    expect(rtAgreementTypeDropDown.first().props().value).toBe(InstitutionMembershipRequirement.DOMAINS);
+  });
+
+  it('should default DOMAINS to ADDRESSES in Controlled Tier requirement', async() => {
+    urlParamsStore.next({
+      institutionId: 'Verily'
+    });
+    const wrapper = component();
+    await waitOneTickAndUpdate(wrapper);
+    expect(wrapper).toBeTruthy();
+
+    const ctEnabledToggle = wrapper.find('[data-test-id="ct-enabled-switch"]').first().instance() as InputSwitch;
```

Comment:
could you add this toggle logic to react-testing-utility ?  It would be great if we could reuse this and call it with a nice signature like `toggleSwitch(switch)`

Reply:
Done I leave the wrapper.find(by id) outside the helper. because I throught now all the test case want to find switch by id? But let me know if I should just make the helper function takes a wrapper and switch `data-test-id`